From 4c500eff9d12ad7fb8e928a4b0ec65246c6977eb Mon Sep 17 00:00:00 2001
From: Jeremy Rocher <jeremy.rocher@intel.com>
Date: Tue, 17 Feb 2015 19:35:17 +0100
Subject: [PATCH] pstore: fix regression on extra_dumper

Fix parameter switch introduced by :
 5e348b4226 Android logger: quick fixup of compilation error

Change-Id: I2cbc009503050413b6709352de444bd278d627ab
Signed-off-by: Jeremy Rocher <jeremy.rocher@intel.com>
---
 fs/pstore/platform.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/pstore/platform.c b/fs/pstore/platform.c
index ce5b341..cd5e4db 100644
--- a/fs/pstore/platform.c
+++ b/fs/pstore/platform.c
@@ -389,7 +389,7 @@ static void pstore_dump(struct kmsg_dumper *dumper,
 					read = len;
 
 				ret = psinfo->write(PSTORE_TYPE_EXTRA, reason, &id, part,
-						oopscount, read, false, psinfo);
+						oopscount, false, read, psinfo);
 
 				if (ret < 0)
 					goto extra_done;
-- 
1.9.1

